packages/extension-messages/src/index.ts

```diff
@@ -32,3 +32,13 @@ export const [shouldBeBlocked, onShouldBeBlockedRequest] = createScopedMessage<
   void,
   boolean
 >('shouldBeBlocked');
+
+export const [
+  getIsUnblockAllWithSingleClick,
+  onGetIsUnblockAllWithSingleClickRequest,
+] = createScopedMessage<void, boolean>('isUnblockAllWithSingleClick');
+
+export const [
+  setIsUnblockAllWithSingleClick,
+  onSetIsUnblockAllWithSingleClick,
+] = createScopedMessage<boolean, boolean>('setUnblockAllWithSingleClick');
```

Comment:
The idea behind `createScopedMessage` generics is to let TS compiler know what types to expect in the `set...` function and what type to expect in return.

In our particular case the `setIsUnblockAllWithSingleClick` exepects a param of boolean and returns nothing. So I'd change it to

```ts
export const [
  setIsUnblockAllWithSingleClick,
  onSetIsUnblockAllWithSingleClick,
] = createScopedMessage<boolean, void>('setUnblockAllWithSingleClick');
```

Reply:
Thank you for your feedback, I'll fix it :)